src/sbvr-api/sbvr-utils.coffee

```diff
@@ -724,6 +724,8 @@ constructError = (e) ->
 		{ status: 500 }
 	.catch UnsupportedMethodError, (err) ->
 		{ status: 405 }
+	.catch permissions.MissingOrInvalidArgumentsError, (err) ->
```

Comment:
This seems to have the same meaning as `uriParser.BadRequestError`, can you at least merge them to one catch block, or even better use a single error for both - in future it'd be nice for hooks to be able to throw a `uriParser.BadRequestError` in order to get a 400 response rather than the default ..404? error, and having multiple equivalent errors would be confusing imo